content/guides/tasks/assignments/update.md

```diff
@@ -0,0 +1,15 @@
+---
+rank: 4
+related_endpoints:
+  - put_task_assignments_id
+related_guides: []
+required_guides: []
+alias_paths: []
+---
+
+# Change a task assignment
+
+To update a task assignment in Box you will need to call the
+[`PUT /tasks/:task_id/assignments`](e://put_task_assignments_id) API.
+
+<Samples id='put_task_assignments_id' />
```

Comment:
I think we might want to split this out into the specific use cases for this endpoint. In this case, you can use it to update the task assignment message, as well as changing the state of the assignment.

Reply:
Are there unique code examples for this I can pull in or will this be new?

Reply:
You'd have to dig into the SDK docs on GH for this. You can add variations to their docs by adding a sample tag as follows:

```
<!-- put_task_assignments_id variation_name -->
```

And in here we can then reference these as follows:

```
<Samples id='put_task_assignments_id' variation='variation_name' />
```

If the samples don't already exist, feel free to instead add them here first using plain code samples and tabs, and then we can submit these to the SDK docs later.

Reply:
A few of the samples already update both the message and the resolution state, so the message guide will duplicate but I'll be pushing this up shortly.

Reply:
I added the samples in-line to the docs, but also opened PRs to the node, java, .net, and python SDKs to support this sample variant. If those get merged in first, I can update the guide to use those.